packages/@react-spectrum/list/src/DragPreview.tsx

```diff
@@ -30,41 +31,44 @@ export function DragPreview(props: DragPreviewProps) {
   let {
     item,
     itemCount,
+    itemHeight,
     provider,
     direction
   } = props;
 
   let isDraggingMultiple = itemCount > 1;
 
   return (
-    <Provider
-      {...provider}
-      UNSAFE_style={{direction}}>
-      <div className={classNames(listStyles, 'react-spectrum-ListViewItem', 'react-spectrum-ListViewItem-dragPreview', {'react-spectrum-ListViewItem-dragPreview--multiple': isDraggingMultiple})}>
-        <Grid UNSAFE_className={listStyles['react-spectrum-ListViewItem-grid']}>
-          <SlotProvider
-            slots={{
-              content: {UNSAFE_className: listStyles['react-spectrum-ListViewItem-content']},
-              text: {UNSAFE_className: listStyles['react-spectrum-ListViewItem-content']},
-              description: {UNSAFE_className: listStyles['react-spectrum-ListViewItem-description']},
-              icon: {UNSAFE_className: listStyles['react-spectrum-ListViewItem-icon'], size: 'M'},
-              image: {UNSAFE_className: listStyles['react-spectrum-ListViewItem-image']},
-              link: {UNSAFE_className: listStyles['react-spectrum-ListViewItem-content'], isQuiet: true},
-              actionButton: {UNSAFE_className: listStyles['react-spectrum-ListViewItem-actions'], isQuiet: true},
-              actionGroup: {
-                UNSAFE_className: listStyles['react-spectrum-ListViewItem-actions'],
-                isQuiet: true,
-                density: 'compact'
-              },
-              actionMenu: {UNSAFE_className: listStyles['react-spectrum-ListViewItem-actionmenu'], isQuiet: true}
-            }}>
-            {typeof item.rendered === 'string' ? <Content>{item.rendered}</Content> : item.rendered}
-            {isDraggingMultiple &&
-              <div className={classNames(listStyles, 'react-spectrum-ListViewItem-badge')}>{itemCount}</div>
-            }
-          </SlotProvider>
-        </Grid>
-      </div>
-    </Provider>
+    <div dir={direction}>
```

Comment:
Ugly diff here: 
- added a wrapper div with `dir={direction}` to get proper RTL styles.

- added `background: 'none'` to provider style to get proper rounded corners 

- added fixed height to listviewitem div

Reply:
Just noticed that adding the div with a different direction gives this warning:

`Language directions cannot be nested. ltr inside rtl.`

Anyone know how to give Provider RTL direction here? Not sure why it would be incorrect